staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -48,7 +48,6 @@ import (
 	// make sure the generated client works
 	_ "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	_ "k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions"
-	_ "k8s.io/apiextensions-apiserver/pkg/client/informers/internalversion"
```

Comment:
Please revert this one.

Reply:
Done.

We keep it here because we put it here `in purpose`, even though line 43 is deleted one day, this will still work.
Do I understand it right?